pkg/lint/runner.go

```diff
@@ -29,6 +29,7 @@ type Runner struct {
 	Log        logutils.Log
 }
 
+//nolint:funlen
```

Comment:
the function is really too long, why not split it?

Reply:
Moved some of the logic for creating new processors out of this function